test/test_dynamic_shapes.py

```diff
@@ -134,7 +134,7 @@ def create_symbolic_tensor(name, arg, shape_env):
     return FakeSymbolicTensor(sym_shapes, sym_strides, arg.dtype, arg.layout, arg.requires_grad, arg.device)
 
 
-CPP_SYMINT_CLASS = type(torch._C.SymIntNode.new_symint(1))
+CPP_SYMINT_CLASS = type(torch.SymIntNode.new_symint(1))
```

Comment:
how come we take the type of an instance instead of directly referring to a class?  I guess its either (a) its hard to refer to a class or (b) we expect different classes depending on which mode this test is run under?

Reply:
I don't know, I just replaced all the code haha.

cc: @Krovatkin